### DISCUSSION TITLE
How do I use visual mode commands in extend mode?

### DISCUSSION THREAD

**Discussion by qwbarch** [mg979/vim-visual-multi#182] [Q&A] [answered]
If I wanted to select the words from line 15-30, here's what I would do:
```
:15
30gg
```
I wanted to try creating cursors from lines 15-30, but this failed:
```
c-n
30gg
```
This simply jumps to line 30 without creating any additional cursors.  
How can I achieve what I'm looking for?

**Comment by mg979** [comment, marked as answer]
<c-n>

Then either `m` operator

    15mj

or `:VMSearch`

    15:VMSearch

or from visual mode

    :h vm-visual-mode

If you wanted to create a column of cursors, you would just do

    15<c-down>
